update

```diff
@@ -11,7 +11,7 @@ const swarmhash = require('swarmhash')
 // This script updates the index files list.js and list.txt in the bin directory,
 // as well as the soljson-latest.js files.
 
-var dirs = ['/bin', '/wasm']
+var dirs = ['/macos', '/linux', '/wasm', '/bin']
```

Comment:
this does not have windows because it does not have a circelci config, right?

Reply:
Yes, exactly. I wanted to take care of windows builds in a separate PR, when I can actually get them.